back/app/services/avatars_service.rb

```diff
@@ -13,11 +13,12 @@ def initialize(participants_service = ParticipantsService.new)
   # @param limit[Integer] Limit the number of users returned. Does not affect the total count
   # @return [Hash] A hash containing the users with avatars and total count of participants
   def avatars_for_project(project, users: User.registered, limit: 5)
```

Comment:
Redid this function as it was always doing double caching, but now it calls in two cached methods

Reply:
Wonder if we could get away with only an hour here though